src/parser/parse_tree_cl.c

```diff
@@ -12910,14 +12910,44 @@ pt_print_insert (PARSER_CONTEXT * parser, PT_NODE * p)
   b = pt_append_varchar (parser, b, r1);
   if (r2)
     {
-      char *class_name = NULL;
-
       b = pt_append_nulstring (parser, b, " (");
 
-      while (class_name = strstr ((char *) r2->bytes, (char *) r1->bytes))
+      if (p->info.insert.hint & PT_HINT_USE_SBR)
 	{
-	  strcpy (class_name, class_name + r1->length + 1);
-	  r2->length = r2->length - r1->length - 1;
+	  char *class_name = NULL;
```

Comment:
recommend: rebuild column_list as below

column_name ref to p->info.insert.attr_list.info.name.original
next column_name ref to p->info.insert.attr_list.next

Reply:
@byungwook-kim Your recommendation is accepted. I appreciate your advice.